conseil-api/src/main/resources/metadata/ethereum.mainnet.conf

```diff
@@ -145,5 +145,120 @@
         }
       }
     }
+    recipts {
```

Comment:
is it correctly spelled? shouldn't be receipts?

Reply:
I've fixed all occurrences in both PRs. What a shame :)